trace/zipkin/zipkin.go

```
@@ -214,7 +226,11 @@ func (z *zipkin) Close() error {
 }
 
 func (z *zipkin) Collect(s *trace.Span) error {
-	z.spans <- s
+	select {
+	case z.spans <- s:
+	default:
```

Return an error rather than logging.